services/omdb/index.js

```diff
@@ -0,0 +1,7 @@
+const axios = require('axios');
+const { requests } = require('../../utils');
+
+exports.requestMovieDetails = id => {
```

Comment:
Why is this just `exports.req---` instead of `module.exports.req---`?

```suggestion
module.exports.requestMovieDetails = id => {
```